api/v1/client.go

```diff
@@ -845,6 +849,173 @@ func (c *storageClient) Account(ctx context.Context, r *http.Request) (*Account,
 	return &a, nil
 }
 
+// Delegations returns a list of delegations.
+func (c *storageClient) Delegations(ctx context.Context, r *http.Request) (*DelegationList, error) {
+	chainID, ok := ctx.Value(ChainIDContextKey).(string)
+	if !ok {
+		return nil, common.ErrBadChainID
+	}
+
+	qb := NewQueryBuilder(fmt.Sprintf(`
+			SELECT delegatee, shares, escrow_balance_active, escrow_total_shares_active
+				FROM %s.delegations
+				JOIN %s.accounts ON %s.delegations.delegatee = %s.accounts.address
+			`,
+		chainID, chainID, chainID, chainID), c.db)
```

Comment:
Can you just do this? Why's do we need to `JOIN` if we're not using any columns from `accounts`?
```suggestion
			SELECT delegatee, shares, escrow_balance_active, escrow_total_shares_active
				FROM %s.delegations
				WHERE delegator = $1::text;
```

and then pass in `chi.URLParam(r, "address")` to the `db.Query` call.